gen/flights/flightplan.py

```diff
@@ -1182,14 +1182,24 @@ def racetrack_for_objective(
         start = end.point_from_heading(heading - 180, diameter)
         return start, end
 
-    @staticmethod
-    def aewc_orbit(location: MissionTarget) -> Point:
+    def aewc_orbit(self, location: MissionTarget) -> Point:
         closest_airfield = location
-        # TODO: This is a heading to itself.
-        # Place this either over the target or as close as possible outside the
-        # threat zone: https://github.com/Khopa/dcs_liberation/issues/842.
         heading = location.position.heading_between_point(closest_airfield.position)
```

Comment:
This is still guaranteed to be due north, which isn't necessarily away from the threat.

You might need to play with shapely a bit here, but what you need to do is find a direction that is at least likely to be safer. The BARCAP planning isn't perfect either and just backs away from the closest enemy airfield. That might accidentally back into another enemy airfield if the mission location is surrounded, of course, but it's better than nothing. You could copy that here.